src/components/common/modal/modals.tsx

```diff
@@ -0,0 +1,46 @@
+import { EModalType, TModalProps } from "src/context-providers";
+import { useModal } from "src/context-providers/display/modal-hook";
+import { EmptyModal } from "./empty-modal";
+import { ScrollableModal } from "./empty-modal.styles";
+
+export const Modals = () => {
+    // =========================================================================
+    // CONST, STATE, REFS
+    // =========================================================================
+    const { modals, hideModal } = useModal();
+
+    // =========================================================================
+    // HELPER FUNCTIONS
+    // =========================================================================
+    const handleBackDropClick = (type: EModalType) => {
+        hideModal(type);
+    };
+
+    const renderModalContent = (modal: TModalProps, index: number) => {
+        switch (modal.type) {
+            case EModalType.BulkEdit:
+            case EModalType.Custom:
+            case EModalType.DiscardChanges:
+                return (
+                    <ScrollableModal
+                        show={!!modal}
+                        onClick={() => handleBackDropClick(modal.type)}
+                    >
+                        <EmptyModal modal={modal} index={index} />
+                    </ScrollableModal>
```

Comment:
no need to render ScrollableModal here 

```suggestion
                        <EmptyModal modal={modal} index={index} />
```

Reply:
fixed in latest push